components/nimbus/src/stateful/behavior.rs

```diff
@@ -569,3 +572,19 @@ impl EventStore {
         Ok(query_type.error_value())
     }
 }
+
+pub fn query_event_store(
```

Comment:
This was moved here out of `src/targeting.rs`. Why didn't you add a new `stateful/targeting.rs` file instead of putting it in here?

Reply:
So while it's primarily _used_ in targeting right now, I moved it here because I felt it more closely aligned with the behavior store as a whole rather than only targeting